src/js/bootstrap-slider.js

```diff
@@ -741,9 +741,21 @@ const windowIsDefined = (typeof window === "object");
 			this.touchmove = this._touchmove.bind(this);
 
 			if (this.touchCapable) {
+				// Test for passive event support
+				var supportsPassive = false;
```

Comment:
could you declare this variable with `let` instead of `var` to scope to nearest enclosing block?